wolfcrypt/src/cryptocb.c

```diff
@@ -1812,7 +1838,7 @@ int wc_CryptoCb_Cmac(Cmac* cmac, const byte* key, word32 keySz,
 
     return wc_CryptoCb_TranslateErrorCode(ret);
 }
-#endif
+#endif /* WOLFSSL_CMAC && !NO_AES */
```

Comment:
Isn't this endif just `/* WOLFSSL_CMAC */`?

Reply:
Good catch.  Updated.